src/libs/actions/Report.js

```diff
@@ -940,10 +940,11 @@ function addAction(reportID, text, file) {
  *  is last read (meaning that the entire report history has been read)
  */
 function updateLastReadActionID(reportID, sequenceNumber) {
-    // If we aren't specifying a sequenceNumber and have no maxSequenceNumber for this report then we should not update
-    // the last read. Most likely, we have just created the report and it has no comments. But we should err on the side
-    // of caution and do nothing in this case.
-    if (_.isUndefined(sequenceNumber) && _.isUndefined(reportMaxSequenceNumbers[reportID])) {
+    // If we aren't specifying a sequenceNumber and have no valid maxSequenceNumber for this report then we should not
+    // update the last read. Most likely, we have just created the report and it has no comments. But we should err on
+    // the side of caution and do nothing in this case.
+    if (_.isUndefined(sequenceNumber)
+        && (!reportMaxSequenceNumbers[reportID] && reportMaxSequenceNumbers[reportID] !== 0)) {
```

Comment:
NAB, but why not use `??`
```
(reportMaxSequenceNumbers[reportID] ?? 0) !== 0
```

Reply:
Here's the result I want...

Only allow the value if `undefined` or `null` but not `0`

and figure that out by looking for something falsy 

`!reportMaxSequenceNumbers[reportID]`

and (`&&`)

that is not `0`

`reportMaxSequenceNumbers[reportID] !== 0`

since if `reportMaxSequenceNumbers[reportID]` was `0` it would be falsy.